components/brave_rewards/browser/publisher_info_database.cc

```diff
@@ -307,17 +307,17 @@ PublisherInfoDatabase::GetPanelPublisher(
   }
 
   sql::Statement info_sql(db_.GetUniqueStatement(
-      "SELECT pi.publisher_id, pi.name, pi.url, pi.favIcon, pi.provider, "
-      "pi.verified, pi.excluded, ai.percent FROM publisher_info AS pi "
-      "LEFT JOIN activity_info AS ai ON pi.publisher_id = ai.publisher_id "
-      "WHERE pi.publisher_id=? AND ((ai.month = ? "
-      "AND ai.year = ? AND ai.reconcile_stamp = ?) OR "
-      "ai.percent IS NULL) LIMIT 1"));
+      "SELECT pi.publisher_id, pi.name, pi.url, pi.favIcon, "
```

Comment:
I know this isn't the original PR; but I would recommend looking at situations where using a view makes sense (instead putting the select statement here). With a view, you can select from it like a table and there's a possibility you can re-use the view in other methods. Kind of nice because you abstract away the joins and basically just have the business logic in here